runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatOverlay.java

```diff
@@ -183,25 +190,54 @@ private String getChangeString(
 		final String prefix = value > 0 ? "+" : "";
 		final String suffix = showPercent ? "%" : "";
 		final String valueString = (int)value == value ? String.valueOf((int)value) : String.valueOf(value);
-		return label + ": " + ColorUtil.wrapWithColorTag(prefix + valueString + suffix, color) + "</br>";
+		return ColorUtil.wrapWithColorTag(prefix + valueString + suffix, color);
 	}
 
-	private String buildStatBonusString(ItemStats s)
+	private String buildStatRow(
+		final String label,
+		final double value,
+		final double diffValue,
+		final boolean inverse,
+		final boolean showPercent)
 	{
-		final StringBuilder b = new StringBuilder();
-		if (config.showWeight())
+		return buildStatRow(label, value, diffValue, inverse, showPercent, false);
+	}
+
+	private String buildStatRow(
+		final String label,
+		final double value,
+		final double diffValue,
+		final boolean inverse,
+		final boolean showPercent,
+		final boolean dontShowBase)
+	{
+		StringBuilder b = new StringBuilder();
+		if (value != 0 || diffValue != 0)
 		{
-			b.append(getChangeString("Weight", s.getWeight(), true, false));
+			String changeStr = getChangeString(diffValue, inverse, showPercent);
+			if (config.alwaysShowBaseStats() && !dontShowBase)
```

Comment:
Only use for dontShowBase is weight, and you are negating it there as well, wouldnt it make more sense to just change the param to showBase and remove the negations?

Reply:
Also empty line above